programming/sr/cheat_sheet.md

```diff
@@ -9,6 +9,14 @@ This page contains a quick guide to the `sr.robot3` API.
 
 For more information, make sure you check the rest of the documentation.
 
+## Import the API
```

Comment:
Very open to better names for this -- obviously we know what this means, but I'm not sure that someone new to programming is going to thing "I need to import the API". Ideally feels like this would use terms that a newbie would think to look for.